platform/pulp/app/models/task.py

```diff
@@ -70,20 +70,23 @@ class Worker(Model):
 
         name (models.TextField): The name of the worker, in the format "worker_type@hostname"
         last_heartbeat (models.DateTimeField): A timestamp of this worker's last heartbeat
+        online (models.BooleanField): Whether is the worker online or not. Default is True.
     """
     objects = WorkerManager()
 
     name = models.TextField(db_index=True, unique=True)
     last_heartbeat = models.DateTimeField(auto_now=True)
+    online = models.BooleanField(default=True)
 
     def save_heartbeat(self):
         """Save a worker heartbeat
 
         Update the last_heartbeat field to now and save it.
 
-        Warning:
+        Warnings:
 
-            Only the last_heartbeat field will be saved. No other changes will be saved.
+            1) Only the last_heartbeat field will be saved. No other changes will be saved.
+            2) If you will try to save via this method new model it will lead to crash.
```

Comment:
I think this line 2) should be converted into a `Raises` statement like this:  https://github.com/BrnoPCmaniak/pulp/blob/bd1f983ab32ba94d0b44e3583baa9ee0a18ee468/platform/pulp/app/models/task.py#L54